### PR TITLE
Update README-ru.md

### PR DESCRIPTION
**Pull request by girshcatt**

### CODE REVIEW

File: README-ru.md [hunk 1]
```diff
@@ -89,7 +90,7 @@
 
 ## 💖 Спонсоры
 
-<p><!-- sponsors --><a href="https://github.com/4kaimar"><img src="https://github.com/4kaimar.png" width="60px" alt="" /></a><a href="https://github.com/EladBG-code"><img src="https://github.com/EladBG-code.png" width="60px" alt="" /></a><!-- sponsors --></p>
+<p><!-- sponsors --><a href="https://github.com/4kaimar"><img src="https://github.com/4kaimar.png" width="60px" alt="" /></a><a href="https://github.com/gordongw"><img src="https://github.com/gordongw.png" width="60px" alt="Gordon" /></a><a href="https://github.com/MayBeNotSimon"><img src="https://github.com/MayBeNotSimon.png" width="60px" alt="" /></a><!-- sponsors --></p>
```
Comment (JunkFood02) [inline README-ru.md:93]: https://github.com/JunkFood02/Seal/pull/1209#issuecomment-1858844372

**Review by JunkFood02 — CHANGES_REQUESTED** [review]

**Comment by girshcatt** [comment]
Do you mean that will be better if we remove sponsors section in readme ru?

**Comment by JunkFood02** [comment]
Yes, see https://github.com/JunkFood02/Seal/pull/1209/commits/9190dd28a9da22be7c7998d120e0084a23aa8353

**Comment by girshcatt** [comment]
I did the changes for sponsor section, merge pr please

**Pull request merged**